frontend/src/components/AddConcepts.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
## Everything in issue complete?
Is this PR no longer a draft; ready to merge?

Let's make sure everything in this issue is either (a) addressed by this PR, (b) will be addressed later, or (c) no longer relevant:
- #601